src/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs

```diff
@@ -427,13 +427,19 @@ SyndicationLink CreateLink(string prefix)
             }
         }
 
-        [Theory]
-        [MemberData(nameof(WriteTo_TestData))]
-        public void Write_HasFeed_SerializesExpected(SyndicationFeed feed, bool serializeExtensionsAsAtom, string expected)
+        [Fact]
+        public void Write_HasFeed_SerializesExpected()
         {
+            const string Expected =
+@"    <channel>
```

Comment:
nit: indentation off?

Reply:
This is a verbatim string literal, which means all whitespace ends up being included, including any tabs used for indentation, and that ends up affecting the test.  I've just copied this string from the theory it had been using.